data/src/main/java/com/pocs/data/model/user/UserDefaultInfoDto.kt

```diff
@@ -1,10 +1,13 @@
 package com.pocs.data.model.user
 
+import android.graphics.Bitmap
+
 data class UserDefaultInfoDto(
     val name: String,
     val email: String,
     val studentId: Int,
     val company: String? = null,
     val generation: Int,
-    val github: String? = null
+    val github: String? = null,
+    val profileImageUrl: Bitmap? = null
```

Comment:
서버가 안드로이드에 제공하는 것은 비트맵이 아닌 이미지의 url 입니다. 
```suggestion
    val profileImageUrl: String? = null
```